doc/cookbook/source/examples/classifier/random_forest.rst

```diff
@@ -0,0 +1,41 @@
+=============
+Random Forest
+=============
+
+Random Forest is an ensemble learning method which implements multiple decision trees during training. It predicts by using a combination rule on the outputs of individual decision trees.
+
+See :cite:`Breiman2001` for a detailed introduction.
+
+-------
+Example
+-------
+
+CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CMulticlassLabels` are created from training and test data file
+
+.. sgexample:: random_forest.sg:create_features
+
+Combination rules to be used for prediction are derived form the :sgclass:`CCombinationRule` class. Here we create a :sgclass:`CMajorityVote` class to be used as a combination rule.
```

Comment:
BTW. Does RF work if no combination rule is supplied?
It should by default, and then you should say here that changing it is optional